libflux/flux-core/src/formatter/mod.rs

```diff
@@ -52,12 +63,18 @@ pub struct Options {
     ///
     /// Default: true
     pub trailing_newline: bool,
+
+    /// How long a line is allowed to be before newlines are forced
+    ///
+    /// Default: 120
```

Comment:
There's been some internal discussion about line length as it affects accessibility, where `100` has been suggested.

Reply:
cc @pierwill

Reply:
Sure, the only reason it was so high is that when I rewrote the formatter, having this line length kept the formatting that existed before (so less noise while getting things merged)

Reply:
Ah ok. Maybe we should change it to 100 as a separate issue.